### PR TITLE
LPS-88163 Folders don't need to be created in Activator any longer

### PR DESCRIPTION
**Pull request by ruben-pulido**

### CODE REVIEW

**Comment by ruben-pulido** [comment]
ci:test:sf <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ruben-pulido** [comment]
ci:test:relevant <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/headless-apio/folder/folder-apio-client-test/src/testIntegration/java/com/liferay/folder/apio/client/test/FolderApioTest.java [hunk 1]
```diff
@@ -101,6 +101,157 @@ public void testCreateFolder() {
 		);
 	}
 
+	@Test
+	public void testCreateSubfolder() {
+		String path = ApioClientBuilder.given(
```
Comment (cgoncas) [inline modules/apps/headless-apio/folder/folder-apio-client-test/src/testIntegration/java/com/liferay/folder/apio/client/test/FolderApioTest.java]: I think we are calling it href

File: modules/apps/headless-apio/folder/folder-apio-client-test/src/testIntegration/java/com/liferay/folder/apio/client/test/FolderApioTest.java [hunk 2]
```diff
@@ -101,6 +101,157 @@ public void testCreateFolder() {
...
+
+		ApioClientBuilder.given(
+		).basicAuth(
+			"test@liferay.com", "test"
+		).header(
+			"Accept", "application/hal+json"
+		).header(
+			"Content-Type", "application/json"
+		).body(
+			"{\"description\":\"Example description 3\",\"name\":\"Example " +
```
Comment (cgoncas) [inline modules/apps/headless-apio/folder/folder-apio-client-test/src/testIntegration/java/com/liferay/folder/apio/client/test/FolderApioTest.java]: Maybe Brian would tell you: Where there is a 3, a 2 and 1  should be exist also.

File: modules/apps/headless-apio/folder/folder-apio-client-test/src/testIntegration/java/com/liferay/folder/apio/client/test/internal/activator/FolderTestActivator.java [hunk 3]
```diff
@@ -41,9 +34,15 @@
 	public static final String CONTENT_SPACE_NAME =
 		FolderTestActivator.class.getSimpleName() + "ContentSpaceName";
 
+	public static final String FOLDER_DESCRIPTION =
```
Comment (cgoncas) [inline modules/apps/headless-apio/folder/folder-apio-client-test/src/testIntegration/java/com/liferay/folder/apio/client/test/internal/activator/FolderTestActivator.java]: I think we don't need it, right?

**Pull request closed without merging**